### PR TITLE
Add 'typdef->typedef'

### PR DESCRIPTION
**Pull request by luzpaz**
Add `typdef->typedef` to code dictionary

### CODE REVIEW

**Review by peternewman — CHANGES_REQUESTED** [review]
This is a typo applicable to code, so it goes in the normal dictionary!

**Comment by luzpaz** [comment]
@peternewman Yikes, that was a mistake on my part. Corrected in d203ba1bf5d

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -31448,6 +31448,7 @@ typcast->typecast
 typcasting->typecasting
 typcasts->typecasts
 typcial->typical
+typdef->typedef
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:31451]: Possibly suggest typed too as another correction.

Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: 53b8a29

**Review by peternewman — APPROVED** [review]

**Pull request merged**